### PR TITLE
Participant record contains only did

### PR DESCRIPTION
**Pull request by Izzzu**
## What this PR changes/adds

This Pr removes all deprecated properties from Participant class and stores in the registry only the participant's did.
CLI was also changed to remove the idsUrl as for now on all the additional properties of the participant should be resolved from did document.

## Why it does that

Participant record served by registration service should only contain a DID URL. Other service URLs should be resolved through DID document.

## Further notes

Because we remove the idsUrl we want to assert only the presence of participant's did in the integration tests. Therefore, in RegistrationApiCommandLineClientTest we use participant did served from mock server instead of nginx. Otherwise the test would be not idempotent as we don't clean up the data after the tests. The clean up would require adding eg. an api endpoint to delete participants, but the delete operation is not in scope.

For the simplicity the RegistrationApiClientTest still uses both dids server from the nginx.

## Linked Issue(s)

Linked to https://github.com/agera-edc/MinimumViableDataspace/issues/174

## Checklist

- [ ] added appropriate tests?
- [ ] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [ ] added/updated relevant documentation?
- [ ] added relevant details to the changelog? (_skip with label `no-changelog`_)
- [ ] formatted title correctly? (_take a look at the [CONTRIBUTING](https://github.com/eclipse-dataspaceconnector/RegistrationService/blob/main/CONTRIBUTING.md#submit-a-pull-request) and [styleguide](https://github.com/eclipse-dataspaceconnector/RegistrationService/blob/main/styleguide.md) for details_)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -16,6 +16,8 @@ rsApi=3.0.0
 faker=1.0.2
 swagger=2.1.13
 jacksonVersion=2.13.1
+okHttp=4.9.3
```
Comment (cpeeyush) [inline gradle.properties:19]: It is already at the top in this file. So can be removed.

Comment (Izzzu) [reply gradle.properties]: removed

File: system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java [hunk 2]
```diff
@@ -38,62 +42,88 @@
...
+
+    @AfterEach
+    void tearDown() {
+        stopQuietly(httpSourceClientAndServer);
+    }
+
     @Test
     void listParticipants() {
+        var did = did();
+        var api = api(did);
```
Comment (paullatzelsperger) [inline system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java:92]: any reason we can't instantiate these two variables in the `@BeforeEach`? seems less repetitive

File: system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java [hunk 3]
```diff
@@ -132,4 +157,20 @@ void getParticipant_notFound() {
                 .isEqualTo(404);
     }
 
+    private void assertIssuedVerifiableCredential(SignedJWT jwt, String clientDid, Instant startTime) throws ParseException {
+        assertThat(jwt.getJWTClaimsSet().getSubject()).isEqualTo(clientDid);
+        assertThat(jwt.getJWTClaimsSet().getIssueTime().toInstant()).isAfter(startTime);
+    }
+
+    @NotNull
+    private RegistryApi api(String did) {
```
Comment (paullatzelsperger) [inline system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java:166]: nit: should be called `createApi` or similar, to indicate what's being done. Same for `did()`.  
Also, those two could be moved to the `RegistrationServiceTestUtils` class as they are duplicated in the command line client test.

File: system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java (hunk 2 above)
Comment (Izzzu) [reply system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java]: good point, moved to @BeforeEach

File: system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java (hunk 3 above)
Comment (Izzzu) [reply system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java]: renamed and moved to RegistrationServiceTestUtils

**Review by paullatzelsperger — APPROVED** [review]

**Review by cpeeyush — APPROVED** [review]

**Pull request merged**